api/src/api/userRouter.ts

```diff
@@ -206,10 +206,8 @@ export const userRouterFactory = (
 
     if (legalStructureHasChanged(oldUserData, userData)) {
       // prevent legal structure from changing if business has been formed
-
```

Comment:
pebble - all your doing here is deleting empty lines, I don't think this file is needed in the PR